app/services/FaciaPress.scala

```diff
@@ -67,6 +91,14 @@ class FaciaPress(val faciaPressQueue: FaciaPressQueue, val configAgent: ConfigAg
             case Success(_) =>
               EnqueuePressSuccess.increment()
           }
+          val fut2 = Future.traverse(paths)(path => faciaPressTopic.publish(PressJob(FrontPath(path), Live, forceConfigUpdate = pressCommand.forceConfigUpdate)))
+          fut2.onComplete {
+            case Failure(error) =>
```

Comment:
Think these lines should be removed too then!

Reply:
True, I missed this. Done now.